README.md

```diff
@@ -1,32 +1,51 @@
-# PhIP-Flow
-
-A general PhIP-Seq short read alignment pipeline using [NextFlow](https://www.nextflow.io/) 
-to produce a dataset containing the 
-_raw alignment counts_ , 
-_sample annotation table,_ and 
-[_peptide annotation table_]() , 
-all merged into an xarray 
-[DataSet](http://xarray.pydata.org/en/stable/api.html#dataset). 
-This dataset organization can subsequently be queried analyzed by using the 
-[phippery](https://github.com/matsengrp/phippery) Python package.
-
-This repository exists primarily to host the bleeding edge pipeline script, and should not need to be cloned unless
-one would like to modify the pipeline. If looking to simply run with the appropriate 
-configuration scripts, 
-one can simply use the Nextflow's build in 
-git aware infrastructure
-```
-$ nextflow run matsengrp/phip-flow/PhIP-Flow.nf -C foo-phip.config -o bar.phip
-```
-
-For example configuration files and more helpful materials, the 
-[Documentation](https://matsengrp.github.io/phippery/Nextflow.html)
-will walk you through examples 
-that can be found in the 
-[Template repository](https://github.com/matsengrp/phip-flow-template)
-
-The pipeline Directed Acyclic Graph is visualized below:
-<p align="center">
-  <img src="dag.png" width="375">
-</p>
+# PHIP-FLOW
+A Nextflow pipeline for Common Phage Immuno-Precipitation Sequencing experiments.
+See the <TODO>
 
+[![nextflow]()]()
+[![Build Status]()]()
+
+## Quickstart 
+
+Install Nextflow by using the following command: 
+
+    curl -s https://get.nextflow.io | bash 
+    
+Download the Docker image with this command (optional) : 
```

Comment:
Instead of listing this step (which Nextflow will take care of anyway with the commands below), it might be more helpful to the user to say something like "Install Docker" with a link to the Docker Desktop setup page.